### ISSUE TITLE
Translation of error messages

### ISSUE THREAD

**Issue by slolo2000** [mholt/PapaParse#348]
Is there any way to translate current error messages?

It would be nice to have these messages in our native language.

**Comment by mholt** [comment]
I totally agree. But I don't have the time to head up this effort right now. See #331.

**Comment by pokoli** [comment]
I think we can use a library like http://guillaumepotier.github.io/gettext.js/ to make all the strings translatable. Of course, a pull request will be very welcome :)

**Comment by andre-f-paggi** [comment]
Hi,

Can I contribute on this?

My ideia is to have separate js files like languages/en-us.js, languages/pt-br.js, etc., and they all will export a global object that will be passed to the config object.

Something like:

`Papa.Languages.Spanish = { "FieldMismatch": { "Muchos campos: esperado {FIELDS_LENGTH} campos pero se han encontrado {THIS_ROW_FIELDS_LENGTH}" } }`

Anf then I would refactor the addError calls in the code to pass the arguments such as FIELDS_LENGTH in the previous example.

My questions are:

- Is this structure ok for you?
- Should this have module exportation for NodeJs too? Or should I use like global variables just like the example above?
- Or do you have other structure for me to follow?

_PS: I can only translate to portuguese and spanish._

Thanks in advance.

**Comment by pokoli** [comment]
Hi @andre-f-paggi, 

Of course, you can contribute on it. Indeed the best is to use an existing library. See [my comment above ](https://github.com/mholt/PapaParse/issues/348#issuecomment-273710015). This libraries usually have a placeholder for passig parameters. 

The steps will be the following: 

1. Add the library and mark the strings as translatable
2. Create translations for the language.

I think this PR should only take care of the first step, and then we should create new Pull Requests for each translations.

**Comment by andre-f-paggi** [comment]
@pokoli,

I thought that those libraries would be an overkill or too heavy sized, but I was wrong since it only takes 3.23 kb and have a nice extensibility. :smiley: 

The only problem is that I was trying to use this library this morning, but unfortunatly there is an error when using it on Node, since there is a bug in the code that breaks it if document variable does not exist. See: [Github Issue](https://github.com/guillaumepotier/gettext.js/issues/7)

Is there another library you suggest me to use?

**Comment by pokoli** [comment]
@andre-f-paggi I've created [a PR that should fix the default value for node](https://github.com/guillaumepotier/gettext.js/pull/10)

Could you confirm (on the PR) that the problem is fixed. 

Having said that, I don't think we should use another library :grinning:

**Comment by andre-f-paggi** [comment]
I guess so.
I will try something here.
